examples/azure-functions-deployment/azure_functions_deployment.bal

```diff
@@ -1,50 +1,16 @@
-import ballerina/uuid;
 import ballerinax/azure_functions as af;
 
-// HTTP request/response with no authentication
+// This functions gets triggered from a HTTP call and returns a processed HTTP output to the caller.
 @af:Function
-public function hello(@af:HTTPTrigger { authLevel: "anonymous" }
-                      string payload)
-                      returns @af:HTTPOutput string|error {
+public function hello(@af:HTTPTrigger { authLevel: "anonymous" } string payload) returns @af:HTTPOutput string|error {
     return "Hello, " + payload + "!";
 }
 
-// HTTP request to add data to a queue
+// This function gets executed from a HTTP call, when it gets executed it will query the blog storage and pick the 
```

Comment:
```suggestion
// This function gets executed by an HTTP call and when it gets executed, it will query the blog storage and pick the 
```
Is it `blob` or `blog`?